common/src/Model/GameImpl.cpp

```diff
@@ -87,7 +87,8 @@ namespace TrenchBroom {
         Game::PathErrors GameImpl::doCheckAdditionalSearchPaths(const IO::Path::List& searchPaths) const {
             PathErrors result;
             for (const auto& searchPath : searchPaths) {
-                if (!IO::Disk::directoryExists(m_gamePath + searchPath)) {
+                const auto absPath = m_gamePath + searchPath;
+                if (!absPath.isAbsolute() || !IO::Disk::directoryExists(absPath)) {
```

Comment:
This is intended to be in this PR right?

Reply:
Yes, a quick drive by fix, otherwise the app crashes when there's no game path set.